src/main.rs

```diff
@@ -118,6 +123,30 @@ fn print_page(path: &Path) -> Result<(), String> {
     Ok(())
 }
 
+/// Set up display pager
+fn configure_pager(args: &Args) {
+    if args.flag_pager {
+        Pager::with_default_pager(PAGER_COMMAND).skip_on_notty().setup();
+    } else {
+        let config = match Config::load() {
+            Ok(config) => config,
+            Err(ConfigError(msg)) => {
+                eprintln!("Could not load config: {}", msg);
+                process::exit(1);
+            }
+            Err(e) => {
+                eprintln!("Could not load config: {}", e);
+                process::exit(1);
+            }
+        };
+
+        if (config.display.pager) {
```

Comment:
No need for parentheses:

```suggestion
        if config.display.pager {
```